spacy/displacy/__init__.py

```
@@ -12,8 +12,15 @@
 IS_JUPYTER = is_in_jupyter()
 
 
-def render(docs, style='dep', page=False, minify=False, jupyter=IS_JUPYTER,
-           options={}, manual=False):
+def render(
+    docs,
+    style="dep",
+    page=False,
+    minify=False,
+    jupyter=IS_JUPYTER,
+    options={},
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)